modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/setup.jsp

```diff
@@ -18,49 +18,24 @@
 
 <%
 String pkccOptions = (String)request.getAttribute("pkccOptions");
+
+Map<String, Object> additionalProps = HashMapBuilder.<String, Object>put(
+	"pkccOptions", pkccOptions
+).build();
 %>
 
 <div id="<portlet:namespace/>messageContainer"></div>
 
-<aui:button id="startRegistration" value="start" />
+<clay:button
+	additionalProps="<%= additionalProps %>"
```

Comment:
Check other instances, this we usually inline now.

Reply:
Ok thanks